extensions/OpenIddict.Validation/OpenIddictValidationExtensions.cs

```diff
@@ -0,0 +1,155 @@
+        {
+            return builder.AddOpenIddictValidation<OpenIddictToken>();
+        }
+
+        /// <summary>
+        /// Adds a new instance of the OpenIddict validation middleware in the ASP.NET Core pipeline.
+        /// </summary>
+        /// <typeparam name="TToken">The type of the Token entity.</typeparam>
+        /// <param name="builder">The authentication builder.</param>
+        /// <returns>The authentication builder.</returns>
+        public static AuthenticationBuilder AddOpenIddictValidation<TToken>([NotNull] this AuthenticationBuilder builder)
+            where TToken : class
+        {
+            return builder.AddOpenIddictValidation<TToken>(OpenIddictValidationDefaults.AuthenticationScheme);
+        }
+        
+        /// <summary>
+        /// Adds a new instance of the OpenIddict validation middleware in the ASP.NET Core pipeline.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <param name="configuration">The delegate used to configure the validation options.</param>
+        /// <returns>The authentication builder.</returns>
+        public static AuthenticationBuilder AddOpenIddictValidation(
+            [NotNull] this AuthenticationBuilder builder,
+            [NotNull] Action<OpenIddictValidationOptions> configuration)
+        {
+            return builder.AddOpenIddictValidation<OpenIddictToken>(configuration);
+        }
+
+        /// <summary>
+        /// Adds a new instance of the OpenIddict validation middleware in the ASP.NET Core pipeline.
+        /// </summary>
+        /// <typeparam name="TToken">The type of the Token entity.</typeparam>
+        /// <param name="builder">The authentication builder.</param>
+        /// <param name="configuration">The delegate used to configure the validation options.</param>
+        /// <returns>The authentication builder.</returns>
+        public static AuthenticationBuilder AddOpenIddictValidation<TToken>(
+            [NotNull] this AuthenticationBuilder builder,
+            [NotNull] Action<OpenIddictValidationOptions> configuration)
+            where TToken : class
+        {
+            return builder.AddOpenIddictValidation<TToken>(OpenIddictValidationDefaults.AuthenticationScheme, configuration);
+        }
+        
+        /// <summary>
+        /// Adds a new instance of the OpenIddict validation middleware in the ASP.NET Core pipeline.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <param name="scheme">The authentication scheme associated with this instance.</param>
+        /// <returns>The authentication builder.</returns>
+        [EditorBrowsable(EditorBrowsableState.Advanced)]
+        public static AuthenticationBuilder AddOpenIddictValidation(
+            [NotNull] this AuthenticationBuilder builder, [NotNull] string scheme)
```

Comment:
Let's remove the overload accepting a `scheme` (we don't support customizing the scheme name of the server handler, so we shouldn't allow replacing the default validation scheme).